homeassistant/components/media_player/samsungtv.py

```diff
@@ -130,6 +133,10 @@ def get_remote(self):
 
     def send_key(self, key):
         """Send a key to the tv and handles exceptions."""
+        if self._is_powering_off and\
+                (key is 'KEY_POWER' or key is 'KEY_POWEROFF') is not True:
```

Comment:
Compare strings with `==` and just prepend `not` to the expression in the parentheses, instead of `is not True`.

Reply:
Change applied.